packages/@expo/cli/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Add improved error message for missing optional imports. ([#35239](https://github.com/expo/expo/pull/35239) by [@EvanBacon](https://github.com/EvanBacon))
 - Estimate Xcode binary path in minimal builds. ([#33415](https://github.com/expo/expo/pull/33415) by [@EvanBacon](https://github.com/EvanBacon))
 - Support GitHub shorthand for templates ([#33383](https://github.com/expo/expo/pull/33383) by [@satya164](https://github.com/satya164))
+- Create `keep.xml` to prevent resource shrinking on adnroid ([#35465](https://github.com/expo/expo/pull/35465) by [@jakex7](https://github.com/jakex7))
```

Comment:
```suggestion
- Create `keep.xml` to prevent resource shrinking on Android ([#35465](https://github.com/expo/expo/pull/35465) by [@jakex7](https://github.com/jakex7))
```